resources/views/dashboard/controllers/events/index.blade.php

```diff
@@ -38,7 +38,12 @@
                             <td width="500px"><a href="/dashboard/controllers/events/view/{{ $e->id }}"><h4>{{ $e->name }}</h4></a></td>
                         @endif
                         <td>{{ $e->date }}</td>
-                        <td>{{ $e->start_time }} - {{ $e->end_time }}z</td>
+                        <td>
+                            {{ $e->start_time }} to {{ $e->end_time }} Zulu
+                            ({{ timeToLocal($e->start_time, Auth::user()->timezone) }} to {{ timeToLocal($e->end_time, Auth::user()->timezone) }} <a style="color:inherit" href="#" data-toggle="tooltip"
+                                                                                                                                                     title="Showing times in {{ Auth::user()->timezone }}. You can change this on your profile."><i
+                                        class="fas fa-info-circle"></i></a>)
```

Comment:
I think we can clean-up the formatting on these timezone displays a bit to be more standardized throughout - seems like each one is a bit different. 
Current: `01:00 to 02:00 Zulu (21:00 to 22:00 )`
Recommended: `01:00 to 02:00 Zulu (21:00 to 22:00 Local)` <- keep the tooltip icon telling them that they can update their profile here